spec/spec_helper.rb

```diff
@@ -33,7 +33,7 @@
 require 'test_app'
 
 module Fixtures
-  TEST_CONFIG = Mutant::Config::DEFAULT.update(reporter: Mutant::Reporter::Trace.new)
+  TEST_CONFIG = Mutant::Config::DEFAULT.with(reporter: Mutant::Reporter::Trace.new)
```

Comment:
`#merge` implies a binary operation of two "equally shaped" objects. I especially thought about this one but than realized when we have an `anima` object on the left side, and a `Hash` instance on the right side: The operation is not commutative and should not be called `#merge`.